packages/manager/apps/telecom/src/app/telecom/telephony/alias/configuration/feature/feature.constants.js

```diff
@@ -8,4 +8,16 @@ export const VALID_SOUND_EXTENSIONS = [
   'wma',
 ];
 
+export const FEATURE_TYPES = {
+  TRUNK: 'trunk',
+  FAX: 'fax',
+  CARRIER_SIP: 'carrierSip',
+  SIP: 'sip',
+};
+
+export const REDIRECT_TYPES = {
+  REDIRECT: 'redirect',
+  DDI: 'ddi',
+};
+
 export default { VALID_SOUND_EXTENSIONS };
```

Comment:
You need to add them in default exports as ewll.
```suggestion
export default { 
  VALID_SOUND_EXTENSIONS,
  FEATURE_TYPES,
  REDIRECT_TYPES,
 };
```

Reply:
Done